pages/data-design/quick-start.mdx

```diff
@@ -0,0 +1,116 @@
+---
+layout: 'guide'
+title: 'Quickstart: Tracking Plan in Avo'
+---
+
+import Link from '../../components/Link';
+
+Managing a tracking plan is hard. You have to make sure it's kept up to date to what is actually being tracked, make sure existing events and properties are reused where relevant, make sure new events and properties match with the naming conventions you have in place, communicating changes and collect feedback from the team, share implementation instructions with developers, and so on.
+
+If you relate to any of the challenges mentioned above you might want to consider switching your tracking plan over to Avo.
+
+The Avo Tracking Plan is a collaborative web app that will make managing your tracking plan less of a headache, with features like <Link passHref href="/audit/index">Tracking Plan audit</Link> to ensure naming conventions are followed, <Link passHref href="/data-design/global-namespace">global event and property namespace</Link> to ensure consistency across your events and properties, <Link passHref href="/workspace/branches">branched collaboration</Link> with <Link passHref href="/workspace/branches#protected-main-branch">approval workflows</Link> and <Link passHref href="/workspace/branches#share-the-implementation-instructions">auto generated implementation instructions</Link>, <Link passHref href="/workspace/integrations#slack">Slack integration</Link> to keep your team in the loop, <Link passHref href="/workspace/tracking-plan/publishing">auto publishing</Link> to keep your downstream schemas in sync, and more!
+
+When you have your tracking plan defined in Avo you can check out our <Link passHref href="/implementation/avo-functions-overview">code generated, type-safe, functions</Link> for developers to increase tracking implementation speed and reliability, and tracking observability with the <Link passHref href="/data-design/start-using-inspector">Inspector</Link> so you know how your _actual_ tracking compares to the tracking plan.
```

Comment:
Link innards also not `a` wrapped

Reply:
Not 100% sure this is required anymore though.

Reply:
I at least assume it is, we haven't changed anything in the setup for a while. Fixed in 5906593